packages/toolbar/README.md

```diff
@@ -0,0 +1,32 @@
+<div align='center'>
+
+<br />
+
+<img src='https://raw.githubusercontent.com/eclipse-theia/theia/master/logo/theia.svg?sanitize=true' alt='theia-ext-logo' width='100px' />
+
+<h2>ECLIPSE THEIA - TOOLBAR EXTENSION</h2>
+
+<hr />
+
+</div>
+
+## Description
+
+The `@theia/toolbar` extension contributes a global toolbar to the application shell. The toolbar supplements the `Command Palette` and allows users to easily add, remove, and rearrange toolbar items to their liking. The toolbar is hidden by default and ships with a default layout which can be overridden by downstream applications. Extenders can also contribute custom widgets to the toolbar through a contribution point. 
+
+## Additional Information
+
+An example toolbar custom widget and layout override can be found in [here](https://github.com/eclipse-theia/theia/tree/master/examples/api-samples/src/browser/toolbar-contribution-example).
```

Comment:
Suggestion: Perhaps we should rename `toolbar-contribution-example` just to `toolbar`? That would be more consistent with the existing naming scheme. Additionally, we could prefix the fixes themselves with `sample-` instead.

Reply:
Agreed, I have renamed these files/examples